applications/OptimizationApplication/tests/test_optimization_info.py

```diff
@@ -3,96 +3,253 @@
 
 # Import KratosUnittest
 import KratosMultiphysics.KratosUnittest as kratos_unittest
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_info import OptimizationData
 from KratosMultiphysics.OptimizationApplication.utilities.optimization_info import OptimizationInfo
+from KratosMultiphysics.OptimizationApplication.responses.response_function import ResponseFunction
+from KratosMultiphysics.OptimizationApplication.responses.mass_response_function import MassResponseFunction
 
-class TestOptimizationInfo(kratos_unittest.TestCase):
-    class TestRoutine(Kratos.Process):
-        def __init__(self):
-            super().__init__()
-
-    def test_AdvanceSolutionStepGet(self):
-        optimization_info = OptimizationInfo()
-        optimization_info.SetBufferSize(3)
+class TestOptimizationData(kratos_unittest.TestCase):
+    def test_AdvanceStep(self):
+        optimization_info = OptimizationData(3)
```

Comment:
optimization_data

Reply:
This is a reminiscence from the past design. I will change the it to `optimization_data`. As explained in the previous PR, these classes needs to stay separate.

Reply:
moved to a different file to avoid confusion.